src/Command/GenerateCommand.php

```diff
@@ -68,7 +68,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $namespace = $input->getOption('namespace');
         if ($namespace) {
             if (!preg_match('/^(?:(?:[[:alnum:]]+|:vendor)\\\\?)+$/', $namespace)) {
-                throw new \InvalidArgumentException('The provided namespace \'%s\' is not a valid namespace!', $namespace);
+                throw new \InvalidArgumentException(sprintf("The provided namespace '%s' is not a valid namespace!", $namespace));
```

Comment:
Let's add some linebreaks and use double quotes:

```suggestion
                throw new \InvalidArgumentException(sprintf(
                    'The provided namespace "%s" is not a valid namespace!', 
                    $namespace
                ));
```